clouddriver-google/src/main/groovy/com/netflix/spinnaker/clouddriver/google/deploy/ops/loadbalancer/DeleteGoogleHttpLoadBalancerAtomicOperation.groovy

```diff
@@ -105,7 +105,7 @@ class DeleteGoogleHttpLoadBalancerAtomicOperation extends DeleteGoogleLoadBalanc
     List<String> listenersToDelete = []
     projectForwardingRules.each { ForwardingRule rule ->
       def proxy = GCEUtil.getTargetProxyFromRule(compute, project, rule)
-      if (GCEUtil.getLocalName(proxy?.getUrlMap()) == urlMapName) {
+      if (proxy?.urlMap && GCEUtil.getLocalName(proxy?.urlMap) == urlMapName) {
```

Comment:
This is determining if the field is present -- `TargetHttp(s)Proxies` have a `urlMap` and `TargetSslProxies` have a `service`. Since we return all three types from the same function (`getTargetProxyFromRule`) we have to do that check.